app/controllers/compute_resources_controller.rb

```diff
@@ -70,6 +80,16 @@ def update
     else
       process_error
     end
+  rescue OVIRT::OvirtException => e
+    Foreman::Logging.exception("Error while updating ovirt resource", e)
+    process_error(
+      error_msg: _('Error while trying to updating resource: %s') % e.message
```

Comment:
nitpick - this should be `...to update resource`

Reply:
fixed, thanks :)